docs/resources/artifactory_remote_docker_repository.md

```diff
@@ -3,6 +3,17 @@
 Provides an Artifactory remote `docker` repository resource. This provides docker specific fields and is the only way to
 get them
 
+### Passwords
+Passwords can only be used when encryption is turned off (https://www.jfrog.com/confluence/display/RTF/Artifactory+Key+Encryption).
+Since only the artifactory server can decrypt them it is impossible for terraform to diff changes correctly.
+
+To get full management, passwords can be decrypted globally using `POST /api/system/decrypt`. If this is not possible,
```

Comment:
Is it possible to use this to mitigate our password issues @alexhung 

Also, why is this documentation change here? What does it have to do with docker and repos?

Reply:
removed from this PR